<!-- Publications Section -->
    <section id="publications" class="bg-light-gray">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <h2 class="section-heading">Publications</h2>
                </div>
            </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
    <p class="text-muted"> Roman Kogay, Taylor B Neely, Daniel P Birnbaum, Camille R Hankel, Migun Shakya, Olga Zhaxybayeva. (2019) Machine-learning classification suggests that many alphaproteobacterial prophages may instead be gene transfer agents. Genome Biology and Evolution, Volume 11, Issue 10, October 2019, Pages 2941–2953. <a href="https://doi.org/10.1093/gbe/evz206">doi:10.1093/gbe/evz206</a>. Code repository: <a href="https://github.com/ecg-lab/GTA-Hunter-v1/">https://github.com/ecg-lab/GTA-Hunter-v1/</a>. </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted"> Daryl Domman, Christopher Ruis, Matthew J Dorman, Migun Shakya, Patrick Sam Guy Chain. (2019). Novel Insights Into the Spread of Enteric Pathogens Using Genomics. Journal of Infectious Diseases. <a href="https://doi.org/10.1093/infdis/jiz220">doi:10.1093/infdis/jiz220</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted"> Ryan Connor, Rodney Brister, Jan P Buchmann, Ward Deboutte, Rob Edwards, Joan Martí-Carreras, Mike Tisza, Vadim Zalunin, Juan Andrade-Martínez, Adrian Cantu, Michael D’Amour, Alexandre Efremov, Lydia Fleischmann, Laura Forero-Junco, Sanzhima Garmaeva, Melissa Giluso, Cody Glickman, Margaret Henderson, Benjamin Kellman, David Kristensen, Carl Leubsdorf, Kyle Levi, Shane Levi, Suman Pakala, Vikas Peddu, Alise Ponsero, Eldred Ribeiro, Farrah Roy, Lindsay Rutter, Surya Saha, Migun Shakya, Ryan Shean, Matthew Miller, Benjamin Tully, Christopher Turkington, Ken Youens-Clark, Bert Vanmechelen, Ben Busby. (2019). NCBI’s Virus Discovery Hackathon: Engaging Research Communities to Identify Cloud Infrastructure Requirements. Genes. <a href=" https://doi.org/10.3390/genes10090714">doi:10.3390/genes10090714</a>. Code repository: <a href="https://github.com/NCBI-Hackathons/VirusDiscoveryProject">https://github.com/NCBI-Hackathons/VirusDiscoveryProject</a>.</p></div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Wafa M Kooli, Thomas Junier, Migun Shakya, Mathilde Monachon, Karen W Davenport, Kaushik Vaideeswaran, Alexandre Vernudachi, Ivan Marozau, Teddy Monrouzeau, Cheryl D Gleasner, Kim McMurry, Reto Lienhard, Lucien Rufener, Jean-Luc Perret, Olha Sereda, Patrick S Chain, Edith Joseph, Pilar Junier. (2019). Remedial treatment of corroded iron objects by environmental Aeromonas isolates. Appl. Environ. Microbiol. <a href="https://doi.org/10.1093/infdis/jiz220">doi:10.1093/infdis/jiz220</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Migun Shakya, Chien-Chi Lo, Patrick SG Chain. (2019). Advances and Challenges in metatranscriptomic analysis. Frontiers in genetics. <a href=" https://doi.org/10.3389/fgene.2019.00904">doi:10.3389/fgene.2019.00904</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Migun Shakya, Shannon M Soucy, Olga Zhaxybayeva. (2017). Insights into origin and evolution of α-proteobacterial gene transfer agents. Virus evolution. <a href=" https://doi.org/10.1093/ve/vex036">doi:10.1093/ve/vex036</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">CW Philipson, Karen Davenport, Logan Voegtly, C Lo, P Li, Yan Xu, Migun Shakya, Regina Z Cer, Kimberly A Bishop-Lilly, Theron Hamilton, Patrick SG Chain. (2017). Brief protocol for EDGE bioinformatics: Analyzing microbial and metagenomic NGS data. bio-protocol. <a href=" https://doi.org/10.21769/BioProtoc.2622">doi:10.21769/BioProtoc.2622</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Maximilian O Press, Andrew H Wiser, Zev N Kronenberg, Kyle W Langford, Migun Shakya, Chien-Chi Lo, Kathryn A Mueller, Shawn T Sullivan, Patrick SG Chain, Ivan Liachko (2017). Hi-C deconvolution of a human gut microbiome yields high-quality draft genomes and reveals plasmid-genome interactions. bioRxiv. <a href="https://doi.org/10.1101/198713">doi:10.1101/198713</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Rosalinda D’Amore, Umer Zeeshan Ijaz, Melanie Schirmer, John G Kenny, Richard Gregory, Alistair C Darby, Migun Shakya, Mircea Podar, Christopher Quince, Neil Hall (2016). A comprehensive benchmarking study of protocols and sequencing platforms for 16S rRNA community profiling. BMC genomics. <a href="https://doi.org/10.1186s12864-015-2194-9">doi:10.1186/s12864-015-2194-9</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Alexander P Hynes, Migun Shakya, Ryan G Mercer, Marc P Grüll, Luke Bown, Fraser Davidson, Ekaterina Steffen, Heidi Matchem, Mandy E Peach, Tim Berger, Katherine Grebe, Olga Zhaxybayeva, Andrew S Lang. (2016). Functional and evolutionary characterization of a gene transfer agent’s multilocus “genome”. Molecular biology and evolution. <a href="https://doi.org/10.1093/molbev/msw125">doi:10.1093/molbev/msw125</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Richard A Hurt Jr, Michael S Robeson II, Migun Shakya, James G Moberly, Tatiana A Vishnivetskaya, Baohua Gu, Dwayne A Elias (2014). Improved yield of high molecular weight DNA coincides with increased microbial diversity access from iron oxide cemented sub-surface clay environments. PLoS One. <a href="https://doi.org/10.1371/journal.pone.0102826">doi:10.1371/journal.pone.0102826</a> </p>
    </div><div class="box-footer"> </div>
    
    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Migun Shakya, Neil Gottel, Hector Castro, Zamin K Yang, Lee Gunter, Jessy Labbé, Wellington Muchero, Gregory Bonito, Rytas Vilgalys, Gerald Tuskan, Mircea Podar, Christopher W Schadt. (2013). A multifactor analysis of fungal and bacterial community structure in the root microbiome of mature Populus deltoides trees. PLoS One. <a href="https://doi.org/10.1371/journal.pone.0102826">doi:10.1371/journal.pone.0102826</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Migun Shakya, Christopher Quince, James H Campbell, Zamin K Yang, Christopher W Schadt, Mircea Podar. (2013). Comparative metagenomic and rRNA microbial diversity characterization using archaeal and bacterial synthetic communities. Environmental microbiology. <a href="https://doi.org/10.1111/1462-2920.12086">doi:10.1111/1462-2920.12086</a> </p>
    </div><div class="box-footer"> </div>

    <div class="box"><div class="box-title"></div><div class="box-body">
        <p class="text-muted">Migun Shakya, Christopher Quince, James H Campbell, Zamin K Yang, Christopher W Schadt, Mircea Podar. (2013). Comparative metagenomic and rRNA microbial diversity characterization using archaeal and bacterial synthetic communities. Environmental microbiology. <a href="https://doi.org/10.1111/1462-2920.12086">doi:10.1111/1462-2920.12086</a> </p>
    </div><div class="box-footer"> </div>

</div>
</section>